### PR TITLE
[[ MergExt ]] Update mergExt pointer

### PR DESCRIPTION
**Pull request by montegoulding**

### CODE REVIEW

File: tests/lcs/mergext/mergext-url.livecodescript [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+   replace "-" with comma in tTestVersion
+   put ",0,0,0,0" after tTestVersion
+   convert tTestVersion to seconds
+   TestAssert "mergext version is a date", the result is empty
+   
+   repeat for each word tEdition in "Community Indy Business"
+      local tURL
+      put builderMergExtUrl(tMergExtVersion,tEdition) into tURL
+      
+      get shell("curl  -s -o /dev/null -IL -w" && quote & "%{http_code}" & quote && tURL)
```
Comment (peter-b) [inline tests/lcs/mergext/mergext-url.livecodescript:47]: Will this actually result in downloading the whole archive, or just checking for its existence?

Comment (montegoulding) [inline tests/lcs/mergext/mergext-url.livecodescript:47]:
```
-I, --head

(HTTP/FTP/FILE) Fetch the HTTP-header only! HTTP-servers feature the command HEAD which this uses to get nothing but the header of a document. When used on an FTP or FILE file, curl displays the file size and last modification time only.
```

**Comment by peter-b** [comment]
@livecode-vulcan review ok 5628515

**Comment by livecode-vulcan** [comment]
💙 review by @peter-b ok 5628515399645f7d447560a9f4a5965f920afc71

**Comment by livecode-vulcan** [comment]
😎 test success 5628515399645f7d447560a9f4a5965f920afc71
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/2622)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/2632)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/2617)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/2633)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/2639)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/2645)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/2657)

**Comment by peter-b** [comment]
@montegoulding I'm not merging this yet because it looks like you want it to go into 8.0.2-rc-2.  If so, you will need to make a new PR for the `release-8.0.2` branch.

**Comment by montegoulding** [comment]
I have cleared the milestone and will discuss with the team.

**Comment by livecodepanos** [comment]
Since the mergExt pointer is updated in `release-8.0.2` branch, and we are going to merge `release-8.0.2` into `develop-8.0` once we release 8.0.2 STABLE (or earlier), I think we can close this PR

**Pull request closed without merging**